src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -41,19 +45,25 @@ const ShareTrajectoryButton = ({
             ) : null}
             <Tooltip
                 title={
-                    isDisabled
+                    !simulariumFile.name
                         ? "Load a model to perform this action"
                         : "Share trajectory"
                 }
                 placement="bottomLeft"
                 color={TOOLTIP_COLOR}
                 align={{ offset: tooltipOffset }}
+                mouseEnterDelay={0.5}
+                trigger={isSharing ? [] : ["hover"]}
```

Comment:
Also consider adding "focus" to the list of triggers so that tab selection of buttons will show the tooltip as well